src/main/kotlin/io/github/mojira/arisa/modules/commands/FixedCommand.kt

```diff
@@ -14,6 +14,14 @@ class FixedCommand {
         if (issue.resolution !in listOf(null, "", "Unresolved")) {
             throw CommandExceptions.ALREADY_RESOLVED.create(issue.resolution)
         }
+        if (issue.affectedVersions.any {
+                it.releaseDate!!.isAfter(issue.project.versions.first {
+                        v -> v.name == version
+                }.releaseDate)
```

Comment:
Might be good to make this more efficient by avoiding the repeated lookup of
```kotlin
issue.project.versions.first { v -> v.name == version }.releaseDate
```
and only calculating it once.